day_4/Day_4_Heads_or_Tails.py

```diff
@@ -0,0 +1,8 @@
+import random
```

Comment:
Looks good. Lets modify the task a little bit. After the original task code please add the code that will generate float number from 0 to 2.00 and will output the "heads" of number less then 1 and "tails" if number is from 1.00 to 2.00